torch/lib/c10d/CMakeLists.txt

```diff
@@ -11,7 +11,8 @@ if(USE_CUDA)
 elseif(USE_ROCM)
   add_definitions(-DUSE_ROCM=1)
   add_definitions(-D__HIP_PLATFORM_HCC__=1)
-  add_definitions(-DHIP_VERSION=${HIP_VERSION_MAJOR})
+  # same definition as in hip/hip_version.h
+  add_definitions(-DHIP_VERSION=(${HIP_VERSION_MAJOR}*100+${HIP_VERSION_MINOR}))
```

Comment:
Can this be defined in some Pytorch files (I recall there is a place where we define miopen/cudnn versions). Internally we don't use Makefile, so this will still break internal ci.

Reply:
I'm just including the hip_version.h file in applicable files and removed the HIP_VERSION setting in make systems - I don't think that was a good strategy since it overrides the HIP included ones with a lower number (it's HIP_MAJOR_VERSION vs HIP_MAJOR_VERSION*100+HIP_MINOR_VERSION)

Reply:
Why not include hip_version.h in a higher-level file, so that we can use it anywhere, without explicitly include it? 

Also -- you mentioned this file is introduced in ROCm 3. Can we add some guard to that include?